src/Orchard.Web/Modules/Orchard.Indexing/Services/IndexingTaskExecutor.cs

```diff
@@ -295,7 +295,7 @@ public class IndexingTaskExecutor : IIndexingTaskExecutor, IIndexStatisticsProvi
             try {
                 if (deleteFromIndex.Count > 0) {
                     _indexProvider.Delete(indexName, deleteFromIndex);
-                    Logger.Information("Added content items to index: {0}", addToIndex.Count);
+					Logger.Information("Deleted content items from index: {0}",  deleteFromIndex.Count);
```

Comment:
No it still has mostly tabs in it. Orchard follows some specific code formatting conventions such as spaces instead of tabs and putting the { on the same line, javascript style, instead of on a new one.

Luckily you can automate this with a free plugin for Visual Studio called [rebracer](https://visualstudiogallery.msdn.microsoft.com/410e9b9f-65f3-4495-b68e-15567e543c58). 

Once installed as you make new edits it will automatically follow these rules but to fix this file you will need to:

  1. install rebracer
  1. open IndexingTaskExecutor.cs
  1. press <kbd>CTRL-K</kbd> + <kbd>CTRL-D</kbd> to apply source formatting